lib/SPIRV/libSPIRV/SPIRVEnum.h

```diff
@@ -135,6 +136,7 @@ template <> inline void SPIRVMap<SPIRVExtInstSetKind, std::string>::init() {
       "NonSemantic.Shader.DebugInfo.100");
   add(SPIRVEIS_NonSemantic_Shader_DebugInfo_200,
       "NonSemantic.Shader.DebugInfo.200");
+  add(SPIRVEIS_Aux_Data, "NonSemantic.AuxData");
```

Comment:
It would be good to be consistent on spelling everywhere: either `AuxData` or `Aux_Data` (personally I prefer the one without `_`).

Reply:
good point, will do